.github/workflows/tests.yaml

```diff
@@ -27,4 +53,3 @@ jobs:
         with:
           # Required: the version of golangci-lint is required and must be specified without patch version: we always use the latest patch version.
           version: v1.29
-          timeout: 5m
```

Comment:
We needed this before for a bit, things were timing out. Any reason to remove it?

Reply:
@dlorenc I assumed it was mistakenly added, as `timeout` isn't a valid argument for `golangci/golangci-lint-action@v2`. I've added [`timeout-minutes`](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepstimeout-minutes) to the lint step, set to 5 minutes.

Reply:
Ah, thanks!